source/common/http/headers.h

```diff
@@ -208,6 +208,11 @@ class HeaderValues {
     const std::string AcceptEncoding{"Accept-Encoding"};
     const std::string Wildcard{"*"};
   } VaryValues;
+
+  struct {
```

Comment:
I would prefer you move this to jwt_authn directory with similar mechanism (register as singleton), this struct is pretty JWT authn specific.

Reply:
Done